charts/gateway-helm/crds/generated/config.gateway.envoyproxy.io_envoyproxies.yaml

```diff
@@ -275,81 +272,85 @@ spec:
+                                          WindowsHostProcessContainers feature flag. Setting this field without the feature
+                                          flag will result in errors when validating the Pod. All of a Pod's containers must
+                                          have the same effective HostProcess value (it is not allowed to have a mix of HostProcess
+                                          containers and non-HostProcess containers).  In addition, if HostProcess is true
+                                          then HostNetwork must also be set to true.
                                         type: boolean
                                       runAsUserName:
-                                        description: The UserName in Windows to run
-                                          the entrypoint of the container process.
-                                          Defaults to the user specified in image
-                                          metadata if unspecified. May also be set
-                                          in PodSecurityContext. If set in both SecurityContext
-                                          and PodSecurityContext, the value specified
-                                          in SecurityContext takes precedence.
+                                        description: |-
+                                          The UserName in Windows to run the entrypoint of the container process.
+                                          Defaults to the user specified in image metadata if unspecified.
+                                          May also be set in PodSecurityContext. If set in both SecurityContext and
+                                          PodSecurityContext, the value specified in SecurityContext takes precedence.
                                         type: string
                                     type: object
                                 type: object
                             type: object
+                          patch:
+                            description: Patch defines how to perform the patch operation
+                              to deployment
+                            properties:
+                              type:
+                                description: |-
+                                  Type is the type of merge operation to perform
+
+                                  By default, StrategicMerge is used as the patch type.
+                                type: string
+                              value:
+                                description: Object contains the raw configuration
+                                  for merged object
+                                x-kubernetes-preserve-unknown-fields: true
+                            required:
+                            - value
+                            type: object
```

Comment:
highlighting the relevant diff in this file since controller-gen update reformatted all the descriptions